src/components/PostDetails.tsx

```diff
@@ -1,106 +1,105 @@
 import React from 'react';
-import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
+import { Loader } from './Loader';
+
+type Props = {
+  post: Post;
+  isLoadingComments: boolean;
+  error: string | null;
+  comments: Comment[];
+  formOpen: (marker: boolean) => void;
+  isFormOpen: boolean;
+  deleteComment: (id: number) => void;
+  addComment: (comment: Comment) => void;
+};
 
-export const PostDetails: React.FC = () => {
+export const PostDetails: React.FC<Props> = ({
+  post,
+  isLoadingComments,
+  error,
+  comments,
+  formOpen,
+  isFormOpen,
+  deleteComment,
+  addComment,
+}) => {
   return (
     <div className="content" data-cy="PostDetails">
       <div className="content" data-cy="PostDetails">
         <div className="block">
-          <h2 data-cy="PostTitle">
-            #18: voluptate et itaque vero tempora molestiae
-          </h2>
-
-          <p data-cy="PostBody">
-            eveniet quo quis laborum totam consequatur non dolor ut et est
-            repudiandae est voluptatem vel debitis et magnam
-          </p>
+          <h2 data-cy="PostTitle">{`#${post.id}: ${post.title}`}</h2>
+          <p data-cy="PostBody">{post.body}</p>
         </div>
 
-        <div className="block">
+        {isLoadingComments ? (
           <Loader />
+        ) : (
+          <div className="block">
+            {error && (
+              <div className="notification is-danger" data-cy="CommentsError">
+                Something went wrong
```

Comment:
Instead of hardcoding the error message 'Something went wrong', consider dynamically displaying the `error` message. This will make the error handling more flexible and informative.